### ISSUE TITLE
Sentry breaks build on Android, unless disabled

### ISSUE THREAD

**Issue by gnprice** [zulip/zulip-mobile#3585]
Attempting a normal release build fails with this error:
```
$ time yarn build:android --stacktrace
yarn run v1.17.3
$ cd android && ./gradlew assembleRelease -Psigned -Psentry --stacktrace

// ... familiar warning noise ...

> Task :app:bundleReleaseJsAndAssets FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:bundleReleaseJsAndAssets'.
> A problem occurred starting process 'command '/home/greg/z/mobile/android/app/build/generated/assets/react/release/index.android.bundle.map''

// ... giant stack-trace ...

Caused by: java.io.IOException: Cannot run program "/home/greg/z/mobile/android/app/build/generated/assets/react/release/index.android.bundle.map" (in directory "/home/greg/z/mobile"): error=2, No such file or directory
        at net.rubygrapefruit.platform.internal.DefaultProcessLauncher.start(DefaultProcessLauncher.java:25)
        ... 7 more
Caused by: java.io.IOException: error=2, No such file or directory
        ... 8 more
```

It's not obvious this is caused by Sentry, but the workaround turns out to be to skip it, by running just `cd android && ./gradlew assembleRelease -Psigned`.

First saw the issue 2019-07-19 when building v25.7.121; it was fine when building v25.6.120 on 2019-06-18.

As a result I've been leaving Sentry out of all Android releases since then. It'd be good to debug the issue and sort it out so we can start using Sentry again.

**Comment by gnprice** [comment]
This is presumably back, now that we've reverted the Sentry upgrade.

**Comment by gnprice** [comment]
Fixed by #3702.
